src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.searchbar {
+  margin: 20px auto 0;
+  width: 100%;
+}
+
+.big-searchbar {
+  height: 70px;
+}
+
+.small-searchbar {
+  height: 42px;
+}
+
+.searchbar-label {
+  position: relative;
+  display: block;
+}
+
+.searchbar-label:after {
+  position: absolute;
+  content: "";
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.searchbar-label-big:after {
+  background-image: url("./images/Search.svg");
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.searchbar-label-small:after {
+  background-image: url("./images/Search.svg");
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big-input {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small-input {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
you can move styles for big input, into the .input selector, and than modificate some values only for small input, the same for all classes,  it will reduce the number of classes